scripts/runBackgroundService.py

```diff
@@ -0,0 +1,38 @@
+# This file is part of rubintv_production.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.rubintv.production.endOfDay import RubinTvBackgroundService
+from lsst.rubintv.production.utils import checkRubinTvExternalPackages
+from lsst.summit.utils.utils import setupLogging
+
+allSkyPngRoot = '/project/rubintv/allsky/'
+moviePngRoot = '/project/rubintv/auxTel_animation/'
+
+setupLogging()
+checkRubinTvExternalPackages()
+
+print('Running RubinTV background catchup service...')
+# TODO: change dryRunForDeletion for deployment, but test like this first
+allSky = RubinTvBackgroundService(allSkyPngRoot=allSkyPngRoot,
```

Comment:
Should these be `argparse` options, so the script doesn't need to be edited to switch modes?

Reply:
These are actually entry points for the k8s pods which run these, so in this case it would mean adding complexity only to have to recode the k8s runners to put in the args correctly (plus all the channels do it like this).

Really, I will probably just remove `dryRun` entirely once I've seen it print out the correct path - I just put it in because launching a new service and letting it run `shutil.rmtree()` on some path without being _really_ sure it's what you want is not a good plan 🙃